src/TimeToLive.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ResellerClub;
+
+use InvalidArgumentException;
+
+class TimeToLive
+{
+    /**
+     * Minimum TTL for DNS records
+     * This is set on the API and throws an API error if not observed.
+     */
+    const MINIMUM_TTL = 7200;
+
+    /**
+     * @var int
+     */
+    protected $value;
+
+    /**
+     * TimeToLive constructor.
+     *
+     * @param int $value
+     *
+     * @throws InvalidArgumentException
+     */
+    public function __construct(int $value)
+    {
+        if ($value < self::MINIMUM_TTL) {
```

Comment:
This doesn't match the exception message, need to change to 
```
if ($value <= self::MINIUM_TTL) {
```

Reply:
A value equal to the minimum TTL is acceptable, so we don't throw an exception in this case.

i.e. we only want to throw an exception if the value is LESS THAN the minimum, which we do.

Reply:
Sorry misread it.